specification/dimensions/tags.md

```diff
@@ -0,0 +1,77 @@
+# Tags
+
+The Tags column represents the set of finalized user-defined and/or provider-defined tags assigned to Tag Sources.  Tags commonly add business context to billing data to identify and accurately allocate charges.
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter user-defined Tag keys or values.
+
+Provider-defined Tags additionally adhere to the following requirements:
+
+* Provider-defined tags MUST be prefixed with a provider-specified tag key prefix.
+* Providers SHOULD publish all provider-specified tag key prefixes within their respective documentation.
+
+## Provider-Defined vs. User-Defined Tags
+
+The following is an example of one user-defined tag and one provider-defined tag, respectively, with tag key, `foo`.  The first tag, which is user-defined, is not prefixed. The second tag is prefixed with marketplace/ which the provider has specified as a reserved tag key prefix.
+
+```json
+    {
+        "foo":"bar",
+        "marketplace/foo": "bar"
+    }
+```
+
+## Finalized Tags
+
+A Tag can either be static or dynamic. If a Tag is static, its value is immutable. If a Tag is dynamic, its value is determined by a set of predefined user or provider rules. A finalized Tag is the final result of any static or dynamic Tag.
```

Comment:
It seems like this is a choice the CSP / generator of FOCUS data has to make, but, it seems sort of implicit. This looks more like a review of how CSPs do it, which doesn't seem spec-appropriate.

Perhaps something more like

> Providers MAY offer multi-tiered tagging taxonomies as part of their management capabilities. In this case, the value of Tags could be the result of applying the taxonomy and precedence rules set by the provider, potentially suppressing visibility of identical values of a tag key that were defined at a lower priority level.

ultimately the Tags end up being just a pile of JSON, and this is just a small window into how a provider ought to set it.

The following section (example) could be framed as a reference implementation for new providers who may not be sure how to implement the tag merging / precedence functionality.

Reply:
Yes, it is up to the CSP/generator on how to they want to or if they want to configure tag processing guidelines, but I think the purpose of this column is not it's not a pile of JSON but a specific subset of that JSON pile and however the process upstream, we expect EXACTLY 1 key and 1 value downstream, or a finalized tag.

Reply:
ok, we can revisit if people end up being confused about the static / dynamic classification, but it's definitely not a sticking point for me.

Reply:
Let's leave for now and revisit after 1.0